internal/test_runner.go

```diff
@@ -57,12 +60,34 @@ func (runner *GuidelineTestRunner) Run() error {
 	if !allPassed {
 		return errors.New("not all tests have passed")
 	}
+
 	return nil
 }
 
-type stdoutPrinter struct {
+type StdoutPrinter struct {
 }
 
-func (p *stdoutPrinter) Print(message string) {
+func (p *StdoutPrinter) Print(message string) {
 	fmt.Println(message)
 }
+
+func (p *StdoutPrinter) PrintTitle(title string) {
```

Comment:
What is a title? On a general interface, there is usually no context on where this is used. I think title is no general concept in this case. I would rename it to `PrintText` or just use `LogInfo`. I'm actually leaning to `LogInfo` since I don't think we need to many different cases for logging

Reply:
As discussed simplified interface and currently have only regular Print, LogWarning (yellow colour) & LogError (red colour).